website_membership_group/static/src/css/website_membership.css

```diff
@@ -0,0 +1,93 @@
+.o_web_memberships_list {
```

Comment:
can you please use existing bootstrap elements. This so changes in the theme are also reflected in the list.